compiler-rt/lib/radsan/tests/radsan_test_interceptors.cpp

```diff
@@ -221,6 +225,83 @@ TEST(TestRadsanInterceptors, freadDiesWhenRealtime) {
   std::remove(temporary_file_path());
 }
 
+class FdFixture : public ::testing::Test {
```

Comment:
Nice idea! I like the cleanliness this brings. Just a nitpick - could we make the fixture name a bit more descriptive, something like `TestInterceptorsRequiringOpenedFileDescriptor`?

Reply:
Updated to use the clang styling for fixtures. On a quick look these are usually short-and-sweet descriptions of the thing under test, ending with the string "Test". `OpenFdTest` is what I went with as the thing needed for the test is an open file descriptor.

```
lldb/unittests/Host/PipeTest.cpp
17:class PipeTest : public testing::Test {

lldb/unittests/Host/SocketAddressTest.cpp
19:class SocketAddressTest : public testing::Test {

lldb/unittests/Host/posix/TerminalTest.cpp
23:class TerminalTest : public ::testing::Test {

lldb/unittests/Host/HostInfoTest.cpp
21:class HostInfoTest : public ::testing::Test {

lldb/unittests/Core/SourceManagerTest.cpp
18:class SourceFileCache : public ::testing::Test {

lldb/unittests/Core/CommunicationTest.cpp
28:class CommunicationTest : public testing::Test {

lldb/unittests/Core/DiagnosticEventTest.cpp
32:class DiagnosticEventTest : public ::testing::Test {

lldb/unittests/tools/lldb-server/tests/TestBase.h
22:class TestBase: public ::testing::Test {

lldb/unittests/Editline/EditlineTest.cpp
237:class EditlineTestFixture : public ::testing::Test {

lldb/unittests/Interpreter/TestCompletion.cpp
32:class CompletionTest : public testing::Test {

lldb/unittests/Interpreter/TestCommandPaths.cpp
28:class VerifyUserMultiwordCmdPathTest : public ::testing::Test {
```

Reply:
Cool - good idea matching existing style. Apologies - I know it's another nitpick - but I think we can still improve on `OpenFdTest`, because it's not the file descriptor that we're testing - we're really testing the interceptors for functions that use a file descriptor. I think we can definitely shorten my suggestion but I think we can still improve. What about `InterceptorsWithOpenFdTest`? I'm happy to make the change if that would help!

Reply:
Will merge first and apply this change as a patch afterwards to avoid cross-fork pain